frontend/src/features/permits/components/permit-list/ApplicationsInProgressList.tsx

```diff
@@ -62,34 +62,29 @@ export const ApplicationsInProgressList = () => {
     },
   ]);
 
-  const applicationsQuery = useQuery({
-    queryKey: [
-      "applicationsInProgress",
-      pagination.pageIndex,
-      pagination.pageSize,
-      sorting,
-    ],
-    queryFn: () =>
-      getApplicationsInProgress({
-        page: pagination.pageIndex,
-        take: pagination.pageSize,
-        orderBy:
-          sorting.length > 0
-            ? [
-                {
-                  column: sorting.at(0)?.id as string,
-                  descending: Boolean(sorting.at(0)?.desc),
-                },
-              ]
-            : [],
-      }),
-    placeholderData: keepPreviousData,
-    refetchOnWindowFocus: false,
-    retry: 1,
+  const applicationsQuery = useApplicationsInProgressQuery({
+    page: pagination.pageIndex,
+    take: pagination.pageSize,
+    sorting: sorting.length > 0
+    ? [
+        {
+          column: sorting.at(0)?.id as string,
+          descending: Boolean(sorting.at(0)?.desc),
+        },
+      ]
+    : [],
   });
 
   const { data, isError, isPending, isFetching } = applicationsQuery;
 
+  useEffect(() => {
+
```

Comment:
`const totalCount = getDefaultRequiredVal(0, data?.meta?.totalItems);`

Reply:
that's cleaner - thanks. I do keep forgetting about `getDefaultRequiredVal` - thanks for the reminder